.github/workflows/DEPLOY_PLAYGROUND_PREVIEW.yml

```diff
@@ -0,0 +1,49 @@
+name: DEPLOY_PLAYGROUND_PREVIEW
+on:
+  pull_request:
+    types: [ labeled, synchronize ]
+jobs:
+  deploy-preview:
+    # check whether the labeled event was deploy-preview || check whether on new commit of PR the label deploy-preview exists
+    if: github.event.label.name == 'deploy-preview' || contains( github.event.pull_request.labels.*.name, 'deploy-preview')
+    runs-on: ubuntu-20.04
```

Comment:
[suggestion] it's better to use a latest LTS (22.04) or even `ubuntu-latest` here.

Reply:
Thanks for the hint! I remember we pinned the version due to some difficulties with firefox (cf. https://github.com/bpmn-io/form-js/pull/384). But this is not a problem in these actions anyway.